packages/block-library/src/post-featured-image/editor.scss

```diff
@@ -6,87 +6,85 @@
 		backdrop-filter: none; // Removes background blur so the overlay's actual color is visible.
 	}
 
-	&.wp-block-post-featured-image {
```

Comment:
While doing some further testing around this block's styles I noticed that were duplicating the block's class selector here. Removing it didn't appear to impact the block visually in either editor and inspecting via devtools still showed all the same styles applied.

As a result, I've cleaned these styles up. Including reducing the specificity on the [button](https://github.com/WordPress/gutenberg/pull/44286/files#diff-5d5782edc25f6b5bb715554f10a066b6433a93fead2a1b99bbe104a6eb5edd47L23) that also didn't need it.

Reply:
This tests well for me, I can also see no difference in the block's styles in either editor 👍